From khali@linux-fr.org Wed Oct 26 12:08:00 2005
Date: Wed, 26 Oct 2005 21:04:12 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Ben Dooks <ben-linux@fluff.org>
Subject: [PATCH 01/16] i2c: Static function fixes, 1 of 4
Message-Id: <20051026210412.5080cf67.khali@linux-fr.org>
Content-Disposition: inline; filename=static-01-i2c-eeprom.patch

From: Ben Dooks <ben-linux@fluff.org>

eeprom_detect is first declared static and then when
the function is actually implemented, there is no static.

Signed-off-by: Ben Dooks <ben-linux@fluff.org>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/chips/eeprom.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/i2c/chips/eeprom.c
+++ gregkh-2.6/drivers/i2c/chips/eeprom.c
@@ -155,7 +155,7 @@ static int eeprom_attach_adapter(struct 
 }
 
 /* This function is called by i2c_probe */
-int eeprom_detect(struct i2c_adapter *adapter, int address, int kind)
+static int eeprom_detect(struct i2c_adapter *adapter, int address, int kind)
 {
 	struct i2c_client *new_client;
 	struct eeprom_data *data;
